.github/workflows/build-test-image.yaml

```
@@ -24,633 +24,708 @@ env:
+  #     - name: Delete Omnistrate Custom Network - AWS
+  #       continue-on-error: true
+  #       uses: ./.github/actions/delete-omnistrate-custom-network
+  #       with:
+  #         username: ${{ secrets.OMNISTRATE_USERNAME }}
+  #         password: ${{ secrets.OMNISTRATE_PASSWORD }}
+  #         custom_network_name: ${{ env.AWS_NETWORK_NAME }}
 
-      - name: Delete Omnistrate Custom Network - AWS
-        continue-on-error: true
-        uses: ./.github/actions/delete-omnistrate-custom-network
+  cleanup-runner:
```

actionlint (1.7.4)

job "cleanup-runner" needs job "test" which does not exist in this workflow

(job-needs)